modules/API/pages/built-in-endpoints.adoc

```diff
@@ -739,148 +739,167 @@ $ curl -X GET 'http://localhost:9000/deleted_vertex_check?threadnum=10&verbose=0
 
 == Authentication
 
-The endpoints in this subsection allow users to create, refresh and delete authentication tokens for requests made to the REST{pp} server. *These endpoints only exist when* xref:user-access:enabling-user-authentication.adoc[*user authentication is enabled*] *on RESTPP endpoints.*
+The endpoints in this subsection allow users to create, refresh and delete authentication tokens for requests made to the REST{pp} server.
+*These endpoints only exist when* xref:user-access:enabling-user-authentication.adoc[*user authentication is enabled*] *on RESTPP endpoints.*
 
```

Comment:
Just take a note here. If we keep the current implementation, we need to add a warning here.

```
[CAUTION]
====
Please be aware to pass parameters in the json payload for `/requesttoken` api from 3.5. If the old API version is used, it could cause information leakage.
====
```

Reply:
Hi Junru, I think I will add this in the release note instead of here. That is a better place to reference older versions and is a better way of getting the attention of existing users.

Reply:
Hi, Lenny. We have changed the implementation in GLE-2819. Once it is merged, it won't be a problem. From my point of view, this warning is unnecessary now.